templates/terraform/examples/resource_policy_full.tf.erb

```diff
@@ -0,0 +1,25 @@
+resource "google_compute_resource_policy" "bar" {
+  provider = "google-beta"
```

Comment:
generally we like to add a provider block in example just in case (and same in other example)
```
provider "google-beta"{
  region = "us-central1"
  zone   = "us-central1-a"
}
```